data/low_income_thresholds.json

```diff
@@ -169,6 +169,47 @@
       }
     }
   },
+  "MI": {
+    "default": {
+      "source_url": "https://www.consumersenergy.com/residential/programs-and-services/payment-assistance",
+      "thresholds": {
+        "1": 22590,
+        "2": 30660,
+        "3": 38730,
+        "4": 46800,
+        "5": 54870,
+        "6": 62940,
+        "7": 71010,
+        "8": 79080
+      }
+    },
+    "mi-consumers-energy": {
```

Comment:
This was difficult to track down. Based on their [terms and conditions](https://www.consumersenergy.com/-/media/CE/Documents/residential/programs-and-services/electric-vehicles/powermidrive-residential-terms-and-conditions.pdf) the only information is:
> In order to qualify as an Income Qualified Customer, the Applicant will be qualified under the Company’s Electric Tariff governing Income Assistance Service Provision and/or Low Income Assistance Credit which will be identified by Consumers Energy Company within its Operations system (SAP)

There weren't any links but I found some information on Consumer Energy's payment assistance programs [here](https://www.consumersenergy.com/residential/programs-and-services/payment-assistance), which has some different programs, one of which said you qualify for assistance if you make less than 150% of the federal poverty level. We may not want to include this though since it's pretty vague